types/plop/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for plop 1.8
+// Project: https://github.com/amwmedia/plop
+// Definitions by: Alex Young <https://github.com/alsiola>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Inquirer, Question, PromptModule } from "inquirer";
+
+declare module "plop" {
```

Comment:
It seems like you should use the [module.d.ts](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) template rather than this structure.

Reply:
+1, almost never use `declare module`.